src/parsers/manifest/dash/common/indexes/get_segments_from_timeline.ts

```diff
@@ -72,17 +72,10 @@ export default function getSegmentsFromTimeline(
 
   const timelineLength = timeline.length;
 
-  // TODO(pierre): use @maxSegmentDuration if possible
-  let maxEncounteredDuration = timeline.length > 0 &&
-                               timeline[0].duration != null ? timeline[0].duration :
```

Comment:
This  code was actually not needed anymore and linked to some previous optimization we do not need anymore.